openbb_terminal/forex/forex_controller.py

```diff
@@ -37,7 +37,8 @@
 forex_data_path = os.path.join(
     os.path.dirname(__file__), os.path.join("data", "polygon_tickers.csv")
 )
-FX_TICKERS = pd.read_csv(forex_data_path).iloc[:, 0].to_list()
+tickers = pd.read_csv(forex_data_path).iloc[:, 0].to_list()
+FX_TICKERS = tickers + [t[-3:] + t[:3] for t in tickers]
```

Comment:
Do we need to add in something for if len(t) == 6.  I am not sure, but are there any 4 symbol fx tickers?

Reply:
may also wanna make this a set because i notice eurusd and usdeur are both in the list, so they would both be duplicated